src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -129,6 +129,13 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             }
             else
             {
+                if (isDotnetIsolatedApp &&
```

Comment:
Discussed offline. We need to make sure we're not logging warnings/errors for supported scenarios.

Portal creation flows will initialize a Function App without a deployed payload, and we want to make sure users are not greeted with an error upon creation. I've discussed some options with @surgupta-msft, leveraging some of the validation work done by @RohitRanjanMS